cloud/linode/cloud.go

```diff
@@ -123,7 +125,7 @@ func newCloud() (cloudprovider.Interface, error) {
 	// create struct that satisfies cloudprovider.Interface
 	lcloud := &linodeCloud{
```

Comment:
The `Initialize` method here calls `newInstances` yet again for the Node Controller - do we want that one to share this `instances` struct, too?

Reply:
Thank you, good catch. Updated the PR to address this as well.